internal/backend/state_connector_impl.go

```diff
@@ -165,33 +145,3 @@ func (sc *stateConnectorImpl) newContextWithMetadata(ctx context.Context) contex
 
 	return ctx
 }
-
-func (sc *stateConnectorImpl) refresh(ctx context.Context, messageIDs []imap.MessageID, mboxIDs ...imap.MailboxID) error {
-	for _, messageID := range messageIDs {
-		message, mboxIDs, err := sc.connector.GetMessage(ctx, messageID)
```

Comment:
could you please also remove `GetMessage` and `GetMailbox` from `connector.Connector`? Those were only needed for refresh

Reply:
They're dangerous, i only reluctantly implemented them before, because they allow gluon to peek at server state before it's ready -- every update should come through the update channel, not directly from GetMessage/GetLabel

Reply:
Done